src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo + "    ____________________________________________________________\n" +
```

Comment:
DukeException class is not used. Might want to look into creating an abstraction for that